examples/5-nextjs/server/index.js

```diff
@@ -0,0 +1,34 @@
+const { ApolloServer } = require('apollo-server-express');
+const { createServer } = require('http');
+const express = require('express');
+const app = express();
+const cors = require('cors');
+
+const { typeDefs, resolvers } = require('./schema');
+
+const PORT = 4000;
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+});
+
+server.applyMiddleware({ app });
+
+const webServer = createServer(app);
+
+const graphqlEndpoint = `http://localhost:${PORT}${server.graphqlPath}`;
+
+app.use(cors());
+app.get(
+  '/',
+  expressPlayground({
```

Comment:
I've got 'an expressPlayground is not defined' when trying to run this. How could I best start the server?